### ISSUE TITLE
AdGuard problem.

### ISSUE THREAD

**Issue by Andrzej9** [Genymobile/gnirehtet#298]
Gnirehtet works great for me, but
opening AdGuard on the phone terminates the connection.
Can I do something about it?
https://drive.google.com/open?id=1sPzy2pFHxDDe7NeRtQ7DJIvsAEs8O4YU

**Comment by rom1v** [comment]
Only one VPN service can be running at any time. I guess AdGuard uses VpnService too.

**Comment by DirtyAngelica** [comment]
Yeah, AdGuard allows using Tor Orbot as HTTP Proxy. Perhaps there is some way Gnirehtet can be used as HTTP Proxy?
